config/rbac/role.yaml

```diff
@@ -38,8 +32,6 @@ rules:
   - create
   - get
   - list
-  - patch
-  - update
```

Comment:
are we sure we never need to update servicemonitors?

Reply:
not the spec section, status yes